### ISSUE TITLE
I believe your documentation diverged from your gem

### ISSUE THREAD

**Issue by tgmerritt** [kollegorna/rspec-api_helpers#4]
Your rails_tutorial_api (which is great and thank you for putting your time and effort into it - it has helped me greatly) has the following documentation:

https://github.com/vasilakisfil/rails_tutorial_api/blob/master/spec/apis/users_spec.rb

![screen shot 2016-12-12 at 16 18 14](https://cloud.githubusercontent.com/assets/1245864/21119111/a34ac11c-c086-11e6-9cd6-874c2f46c874.png)

But the "it_returns_resources" method doesn't appear to exist in the most recent version of your gem.  I get:

```
/Users/tgmerritt/fh_api/spec/apis/users_spec.rb:13:in `block (2 levels) in <top (required)>': undefined method `it_returns_resources' for RSpec::ExampleGroups::ApiV1UsersController::Index:Class (NoMethodError)
```

And it appears that this method has been renamed:

```
it_returns_collection_size
it_returns_embedded_size
```

I can't tell which one - but based on my gut about what this method should do, and the documentation about these two methods, it's likely one of them.

**Comment by vasilakisfil** [comment]
Hi. Yes that's true I am sorry. I wasn't following very much semantic version.

I just published a new version (1.0.0) which has detailed documentation. From now on I intend to keep the semantic version. Sorry for any troubles!

Also a new version of the blog post will be published before Christmas for Rails 5 I 'll keep you posted!

**Comment by vasilakisfil** [comment]
Closing..
